clustering/infinispan/spi/src/main/java/org/wildfly/clustering/infinispan/spi/persistence/IndexedKeyFormatMapper.java

```diff
@@ -49,7 +49,7 @@ public IndexedKeyFormatMapper(List<? extends KeyFormat<?>> keyFormats) {
             this.indexes.put(this.keyFormats.get(i).getTargetClass(), i);
         }
         // Determine number of characters to reserve for index
-        this.padding = (int) (Math.log(this.keyFormats.size() - 1) / Math.log(HEX_RADIX)) + 1;
+        this.padding = (int) (Math.log( (double) this.keyFormats.size() - 1) / Math.log(HEX_RADIX)) + 1;
```

Comment:
you should also cast Math.log(HEX_RADIX) since HEX_RADIX is an int

Reply:
NP to fix, but Math.log(HEX_RADIX) does not contain a calculation. "this.keyFormats.size() - 1" must be calculated as double.